edc-tests/edc-dataplane/cloud-transfer-tests/src/test/java/org/eclipse/tractusx/edc/dataplane/transfer/test/S3ToS3Test.java

```diff
@@ -0,0 +1,285 @@
+        assertThat(b1.sdkHttpResponse().isSuccessful()).isTrue();
+        // upload test file in provider
+        var putResponse = providerClient.putObject(PutObjectRequest.builder().bucket(PROVIDER_BUCKET_NAME).key(FILE_NAME).build(), TestUtils.getFileFromResourceName(TESTFILE_JSON).toPath());
+        assertThat(putResponse.sdkHttpResponse().isSuccessful()).isTrue();
+
+        // create bucket in consumer
+        var b2 = consumerClient.createBucket(CreateBucketRequest.builder().bucket(CONSUMER_BUCKET_NAME).build());
+        assertThat(b2.sdkHttpResponse().isSuccessful()).isTrue();
+
+        // initiate data flow request
+        var request = createFlowRequest();
+        var url = "http://localhost:%s/control/transfer".formatted(PROVIDER_CONTROL_PORT);
+
+        given()
+                .when()
+                .baseUri(url)
+                .contentType(ContentType.JSON)
+                .body(request)
+                .post()
+                .then()
+                .statusCode(200);
+
+
+        await().pollInterval(Duration.ofSeconds(2))
+                .atMost(Duration.ofSeconds(60))
+                .untilAsserted(() -> assertThat(listObjects(consumerClient, CONSUMER_BUCKET_NAME))
+                        .isNotEmpty()
+                        .contains(FILE_NAME)
+                        .anyMatch(c -> c.endsWith(COMPLETION_MARKER)));
+    }
+
+    @Test
+    void transferFile_targetContainerNotExist_shouldFail() {
+        // create bucket in provider
+        var b1 = providerClient.createBucket(CreateBucketRequest.builder().bucket(PROVIDER_BUCKET_NAME).build());
+        assertThat(b1.sdkHttpResponse().isSuccessful()).isTrue();
+        // upload test file in provider
+        var putResponse = providerClient.putObject(PutObjectRequest.builder().bucket(PROVIDER_BUCKET_NAME).key(FILE_NAME).build(), TestUtils.getFileFromResourceName(TESTFILE_JSON).toPath());
+        assertThat(putResponse.sdkHttpResponse().isSuccessful()).isTrue();
+
+        // do not create bucket in consumer -> will fail!
+
+        // initiate data flow request
+        var request = createFlowRequest();
+        var url = "http://localhost:%s/control/transfer".formatted(PROVIDER_CONTROL_PORT);
+
+        given()
+                .when()
+                .baseUri(url)
+                .contentType(ContentType.JSON)
+                .body(request)
+                .post()
+                .then()
+                .statusCode(200);
+
+
+        // wait until the data plane logs an exception that it cannot transfer the file
+        await().pollInterval(Duration.ofSeconds(2))
+                .atMost(Duration.ofSeconds(10))
+                .untilAsserted(() -> verify(DATAPLANE_RUNTIME.getContext().getMonitor()).severe(startsWith("Error writing the %s object on the %s bucket: The specified bucket does not exist".formatted(FILE_NAME, CONSUMER_BUCKET_NAME)),
+                        isA(NoSuchBucketException.class)));
```

Comment:
Overlong arrow function without extra formatting makes it hard to read.
Maybe pull out the Error string into a variable?